spec/plans/import.fmf

```diff
@@ -91,8 +91,33 @@ description: |
     Inheriting or overriding remote plan config with local plan
     steps might be possible in the future but currently is not
     supported. The imported plan can be modified in only two ways.
-    First way is via environment variables. Variables defined
-    in the plan override any variables defined in the remote plan.
+    First way is via environment and context variables. Imported plan
+    inherits all environment and context variables from the importing
+    plan. This behavior can be changed by setting the ``inherit-context``
+    or ``inherit-environment`` options:
+
+    .. code-block:: yaml
+
+       /some-plan:
+         context:
```

Comment:
Let's stick with the 4-space indentation we're using across `fmf` config examples.